tests/unit/models/gpflow/test_utils.py

```diff
@@ -78,6 +81,50 @@ def test_gaussian_process_deep_copyable(gpflow_interface_factory: ModelFactoryTy
     npt.assert_array_compare(operator.__ne__, variance_f_copy_updated, variance_f_copy_updated2)
 
 
+def test_gaussian_process_tf_saved_model(gpflow_interface_factory: ModelFactoryType) -> None:
```

Comment:
Why is this in `test_utils` and not `test_models` for consistency with the gplux and keras cases?
Moreover, there's a lot of machinery with these tests that can be shared across models, perhaps collapse these into parametrised tests at the `test.unit.models.interfaces` level?

Reply:
Agree, though I think the (significant) refactoring should be in a separate PR. I put this in `test_utils` because that's where the deepcopy tests were, but both should probably go in `test_models`. And the interface tests are currently model agnostic (using toy models when necessary) but there is definitely scope for sharing lots more code.